### PR TITLE
Rename bq scripts path views to view

### PR DESCRIPTION
**Pull request by mservidio**
- Rename bq scripts path views to view.
- Case updates for sql scripts.
- Rename label from cds_managed to datashare_managed.
- Markdown renames from cds -> ds.
- Comment renames from cds -> Datashare.

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### CODE REVIEW

File: shared/README.md [hunk 1]
```diff
@@ -52,25 +52,25 @@ Set your **PROJECT\_ID** if you have not already:
 
 Set the **SERVICE\_ACCOUNT\_NAME** environment variable(s):
 
-    export SERVICE_ACCOUNT_NAME=cds-cloud-build-mgr;
+    export SERVICE_ACCOUNT_NAME=ds-cloud-build-mgr;
```
Comment (mservidio) [inline shared/README.md:55]: also have to rename in cloudbuild.yaml and setup CI to support the change

File: api/v1alpha/src/lib/config.js [hunk 2]
```diff
@@ -19,9 +19,9 @@
 var config = {};
 
 config.verboseMode = false;
-config.productName = 'CDS';
+config.productName = 'Datashare';
 config.productVersion = '0.0.1';
-config.cdsManagedLabelKey = 'cds_managed';
+config.cdsManagedLabelKey = 'datashare_managed';
```
Comment (phriscage) [inline api/v1alpha/src/lib/config.js:24]: Is there a migration path for customers that are using the old label key? i.e. Do we need to support both label keys?

Comment (mservidio) [reply api/v1alpha/src/lib/config.js:24]: Until we publish a v1 public on marketplace I wasn't planning to add any data migration. There's more than just this, Ie table schema changes etc.

Comment (phriscage) [reply api/v1alpha/src/lib/config.js:24]: ack. We can circle back for a migration strategy for v1.

**Pull request merged**